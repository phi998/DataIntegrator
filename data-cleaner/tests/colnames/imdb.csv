Title,Title,Title,Year,Duration (in hours),Earnings,Earnings,Duration (in hours),Duration (in hours),Duration (in hours),Plot,Plot,Director,Actors,Actors,Earnings,Rating,Actors,Actors,Earnings,Earnings,Earnings,Rating,Title,Rating,Rating,Duration (in hours),Title,Duration (in hours),Unknown,Actors,Director,Director,Plot,Plot,Plot,Plot,Title,Actors,Earnings,Rating,Duration (in hours),Duration (in hours),Actors,Year,Duration (in hours),Unknown,other,Year,Title,Title,Earnings,Earnings,Earnings,Year,Earnings,Audio,Unknown,Unknown,Title,Unknown,Unknown,Unknown
Title,Title,Title,Year,Duration (in hours),Earnings,Duration (in hours),Score,Duration (in hours),Duration (in hours),Plot,Title,Director,Directors,Actors,Earnings,Score,Actors,Actors,Earnings,Score,Score,Score,Title,Score,Score,Score,Title,Score,Unknown,Actors,Director,Director,Plot,Plot,Plot,Duration (in hours),Title,Title,Score,Score,Score,Score,Actors,Year,Score,Unknown,Title,Year,Title,Title,Earnings,Earnings,Earnings,Year,Earnings,Sound,Unknown,Unknown,Title,Unknown,Unknown,Unknown
Title,Title,Title,Year,Duration (in hours),Earnings,Earnings,Duration (in hours),Duration (in hours),Duration (in hours),Plot,Title,Director,Directors,Actors,Earnings,other,Director,Actors,Earnings,Earnings,Earnings,Rating,Title,Rating,Rating,Duration (in hours),Title,Duration (in hours),Unknown,Actors,Director,Director,Plot,Duration (in hours),Plot,Plot,Title,Actors,Earnings,Title,Duration (in hours),Duration (in hours),Unknown,Year,Duration (in hours),Plot,Unknown,Year,Title,Production Companies,Earnings,Earnings,Earnings,Year,Earnings,Audio Formats,Unknown,Unknown,Title,Watchlist,Unknown,Unknown
