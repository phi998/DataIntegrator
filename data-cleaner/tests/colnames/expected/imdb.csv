Title,Title,Title,Year,Duration (in hours),Earnings,.,.,.,.,Plot,Plot,Director,Directors,Actors,Earnings,.,Directors,Actors,.,.,.,Rating,Rating,Prize,Rating,.,Title,.,.,Actors,Director,Director,Plot,Plot,Plot,Plot,.,Actors,Earnings,Rating,.,.,.,.,Duration (in hours),.,.,Year,Title,Production Companies,Earnings,Earnings,Earnings,Year,Earnings,Audio,.,.,.,.,.,.
